homeassistant/helpers/template.py

```diff
@@ -2113,6 +2115,42 @@ def as_timedelta(value: str) -> timedelta | None:
     return dt_util.parse_duration(value)
 
 
+def merge_response(
+    value: ServiceResponse, sort_by: str | None = None, single_key: str | None = None
+) -> list[Any]:
+    """Merge service responses into single list.
+
+    Checks that the input is a correct service response:
+    {
+        "entity_id": {str: Any},
+    }
+    Returns empty list by default.
```

Comment:
The function returns the empty list *in case of any error*, is that wanted instead of erroring out? Why don't we want to blow up instead?

Reply:
I changed this so it blows.